templates/typescript_gapic/package.json.njk

```diff
@@ -41,14 +41,17 @@ limitations under the License.
     "mv": "^2.1.1",
     "ncp": "^2.0.0",
     "tmp": "^0.1.0",
-    "typescript": "^3.7.0"
+    "typescript": "^3.7.0",
+    "jsdoc": "^3.5.5",
```

Comment:
nit: can we try to keep the alphabetical order of the dependencies? `npm` does that.

Reply:
sure, run `npx sort-package-json` to sort the dependencies, did not know there's a way to sort without extra package. Thanks!